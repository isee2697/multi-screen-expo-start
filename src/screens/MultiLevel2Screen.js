import React from 'react';
import { ScrollView, Text, View } from 'react-native';
import { utilStyles } from '../api/constants';

const MultiLevel2Screen = () => (
  <View style={utilStyles.container}>
    <ScrollView contentContainerStyle={utilStyles.contentContainer}>
      <Text style={utilStyles.text}>Multi Screens :: Level 2 Content Area</Text>
    </ScrollView>
  </View>
);

MultiLevel2Screen.navigationOptions = {
  headerTitleStyle: {
    flex: 1,
    textAlign: 'center'
  },
  title: 'Level 2'
};

export default MultiLevel2Screen;
