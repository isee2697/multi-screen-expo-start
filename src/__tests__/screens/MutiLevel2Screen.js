import React from 'react';
import { shallow } from 'enzyme';

import MultiLevel2Screen from '../../screens/MultiLevel2Screen';

const navigation = { navigate: jest.fn() };
const screen = shallow(<MultiLevel2Screen navigation={navigation} />);

describe('<MultiLevel2Screen />', () => {
  it('renders correctly', () => {
    expect(screen).toMatchSnapshot();
  });
});
