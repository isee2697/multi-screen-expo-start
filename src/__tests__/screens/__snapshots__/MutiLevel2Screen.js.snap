// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultiLevel2Screen /> renders correctly 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <ScrollViewMock
    contentContainerStyle={
      Object {
        "alignItems": "center",
        "paddingTop": 30,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#1d2d3a",
        }
      }
    >
      Multi Screens :: Level 2 Content Area
    </Text>
  </ScrollViewMock>
</Component>
`;
